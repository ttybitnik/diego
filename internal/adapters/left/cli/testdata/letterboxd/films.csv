Date,Name,Year,Letterboxd URI
2021-12-14,Sans Soleil,1983,https://boxd.it/28B8
2021-12-14,Dekalog,1989,https://boxd.it/bv4u
2021-12-14,Blade Runner,1982,https://boxd.it/2bcA