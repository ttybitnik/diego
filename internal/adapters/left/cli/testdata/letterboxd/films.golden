- name: Sans Soleil
  year: "1983"
  url: https://boxd.it/28B8
  date: "2021-12-14"
- name: Dekalog
  year: "1989"
  url: https://boxd.it/bv4u
  date: "2021-12-14"
- name: Blade Runner
  year: "1982"
  url: https://boxd.it/2bcA
  date: "2021-12-14"
